apps/src/applab/applab.js

```diff
@@ -677,6 +678,7 @@ Applab.init = function(config) {
     isSubmittable: !!config.level.submittable,
     isSubmitted: !!config.level.submitted,
     librariesEnabled: !!config.level.librariesEnabled,
+    aiEnabled: !!config.level.aiEnabled || !!config.level.isProjectLevel,
```

Comment:
Just curious: does this mean all project levels are AI enabled?

Reply:
It means that new App Lab projects will be AI enabled.

Reply:
rather than using `config.level.isProjectLevel`, we'll want to add `ai_enabled` to the level file for new applab projects like we do with `libraries_enabled`: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/scripts/levels/New%20App%20Lab%20Project.level#L34

Reply:
^ 🤦🏻‍♀️ you already did that! you should be able to remove `!!config.level.isProjectLevel` in that case since project levels will have this configuration enabled by default